### DISCUSSION TITLE
Arts Festival Organizer - Need 150+ shows to be easily searchable online

### DISCUSSION THREAD

**Discussion by TorontoFringe** [pair-cg/general-contributions#57] [User Stories]
"As a festival with tons of diverse content, I need to have all the various details about the shows (genre, length, age range, access needs, trigger warnings, etc.) be easily searchable online so that audiences can make the most informed choices and enjoy their experience."

# 1. Need:
_I need a simple way to convey the different show variables to audiences online beyond the scope of our website that already has robust search/filter systems._

# 2. Action:
_Gather show data from festival artists in an appropriate way, edit/process the data, have the show data flow through to Google or other platforms in a bulk way so that people can search on various platforms._

# 3. Desired Outcome:
_Audiences can find what they need more easily so they end up seeing more shows_

# 4. Perspective:
_Staff member of a large arts festival_

# 5. Stakeholder Group:
_Arts Organization_

# 6. Additional info:
_These steps will need to be undertaken by staff members who have very limited knowledge of code or internet platforms._

**Comment by mariel-marshall** [comment]
Hey @TorontoFringe this is great! Thanks so much for including it. With the upcoming festival becoming digital, it would be useful to know more about all of the specific show variables. For example with Bake Your Heart Out with Morro & Jasp ([https://fringetoronto.com/sites/default/files/2021%20Fringe%20NS%20Program%20FINAL%20r1%20web.pdf](https://fringetoronto.com/sites/default/files/2021%20Fringe%20NS%20Program%20FINAL%20r1%20web.pdf)) I see: 

Basic Production Profile:
**title of production:** Bake Your Heart Out with Morro & Jasp
**description of production:** Join celebrity chefs Morro and Jasp (Dora and Gourmand Cookbook Award Winners, and publishers of the cookbook Eat Your Heart Out with Morro & Jasp) for an exclusive Fringe fundraiser event.Our favourite clown sisters are attempting to bake all their troubles away and get through isolation from their respective homes, and you’re invited. This Zoom-based, interactive, and participatory baking extravaganza will lead the audience through a few delectable recipes that will fill your belly and warm your hearts. There will be mess, mayhem, guided baking, and hopefully magic (well at least the clown kind). As Morro said once: “If you can’t cook, do it anyways”.
**Date of performance:** SATURDAY, JANUARY 23
**Time of performance:** 8:30pm
**format of remote attendance mode:** live-stream
**captions:** Auto-Transcribed Captions 
**audience attendance mode:** remote / online
**type of performance:** fundraiser 
**proceed information:** All proceeds from this fundraiser performance support the ongoing operations of the Toronto Fringe.

**tickets / packages** 
**ticket offer 1 title:** Bake Your Heart Out VIP Package
**ticket 1 description** Your VIP Package invites you to participate in the interactive Zoom event. You will also receive a home-delivered package with a signed copy of the cookbook Eat Your Heart Out with Morro & Jasp and ready-to-bake cookie dough, courtesy of Courage Cookies.
**offer 1 price:** $60 CAD
**ticket 1 link url:** ???

**ticket offer 2 title:** BAKE YOUR HEART OUT ALL-DIGITAL PACKAGE 
**offer 2 description:** Your All-Digital Package invites you to watch the interactive Zoom event as a spectator only. Plus we will email you an exclusive cookie recipe from Morro & Jasp.
offer 2price: $30 CAD
**ticket 1 link url:** ???

**These aren't captured but could be added:**
**discipline:** theatre
**genre:** comedy
**style:** clown
**duration of production:**  ???
**age range:** ???  
**trigger warnings:** ???

What else is missing @TorontoFringe, especially for the upcoming Digital Festival?

**Reply by TorontoFringe** [reply]
This is a good question. Since the art is now online, we may need to get into what the medium is - this show above was live-streamed but many shows this summer will be pre-recorded video, audio, or even some of the art will be written PDFs. So media type becomes important. Also the paywall we have been using means that there are no hard tickets to access. There is a pass that allows you to get behind the paywall. Like with a news organization, you pay then you are behind the paywall and can explore various tiers of the digital art. So instead of ticket links something like paywall options?

**Comment by ybressler** [comment]
_Q's for @TorontoFringe:_
* Does the Toronto Fringe have an archive / database of past productions?
  * Is it publically available?
  * Is it internally available?
* Does the Canadian Fringe have an archive / database of:
  * Past productions? 
  * Past Fringe festivals?

**Reply by TorontoFringe** [reply]
Alas we do not have a digital archive. We have scanned versions of printed programs. We have not been archiving our website show listings after each festival (unfortunately.) As far as I know, the Canadian Association of Fringe Festivals (CAFF) does not keep an archive either. Each festival may have their own way of archiving, and it's something I could inquire about.
